dcpy/connectors/socrata/publish.py

```diff
@@ -120,28 +120,30 @@ def _column_update_endpoint(self):
                 )
 
             @property
-            def column_names(self) -> set[str]:
-                return {c["field_name"] for c in self.soc_output_columns}
+            def column_names(self) -> list[str]:
```

Comment:
Is there reason we'd want this here and not have the underlying `self.soc_output_columns` have the same ordering?

Reply:
[looks like](https://github.com/NYCPlanning/data-engineering/blob/dm-cpdb-socrata/dcpy/connectors/socrata/publish.py#L166-L170) when we actually push column metadata we'd probably rather have them in the manual order they're set to in socrata (ideally the same order as in `metadata.yml`) rather than sorted alphabetically

this sort was really just for logging QOL